packages/devextreme/scss/widgets/generic/gallery/_index.scss

```diff
@@ -102,6 +102,14 @@
 }
 
 .dx-state-focused.dx-gallery {
+  border-width: 1px;
```

Comment:
same styles for diff themes, mb we can use some kind of mixin here?

Reply:
as far as border color is defined for different themes separetely, I don't think we should do this

we can move only border-width and border-style to common styles, but I don't think it's a good idea to separate styles of the same border